### PR TITLE
Internal: Add What's New icon inside the Top Bar [ED-13681]

### PR DESCRIPTION
**Pull request by KingYes**

### CODE REVIEW

File: modules/notifications/assets/js/components/editor-v2.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	const { mainMenu } = window.elementorV2.editorAppBar;
+
+	mainMenu.registerLink( {
+		id: 'app-bar-menu-item-whats-new',
+		priority: 1,
+		useProps: () => {
+			return {
+				title: __( "What's New", 'elementor' ),
+				icon: GiftIcon,
+				priority: 5,
```
Comment (rotemee) [inline modules/notifications/assets/js/components/editor-v2.js:15]: Is it needed?

File: modules/notifications/module.php [hunk 2]
```diff
@@ -41,31 +41,42 @@ public function __construct() {
...
+	public function enqueue_editor_scripts() {
+		$deps = [
+			'elementor-editor',
+			'elementor-v2-ui',
+			'elementor-v2-icons',
+			'elementor-v2-query',
+			'wp-i18n',
+		];
+
+		$is_editor_v2 = current_action() === 'elementor/editor/v2/scripts/enqueue';
```
Comment (rotemee) [inline modules/notifications/module.php:59]: What is it current_action? (can't find it in the code).

File: modules/notifications/assets/js/components/editor-drawer.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+	useEffect( () => {
+		document.body.classList.remove( 'e-has-notification' );
+	}, [ isRead ] );
+
+	return (
+		<WhatsNew
+			isOpen={ isOpen }
+			setIsOpen={ setIsOpen }
+			setIsRead={ setIsRead }
```
Comment (rotemee) [inline modules/notifications/assets/js/components/editor-drawer.js:20]: Why do you need it reactive? (if you also don't read the isRead value nowhere), instead you can just use it as an event and remove the isRead state:
```suggestion
			markAsRead={ () =>  document.body.classList.remove( 'e-has-notification' ) }
```

File: modules/notifications/assets/js/components/editor-on-button-clicked.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+export const editorOnButtonClicked = () => {
+	if ( ! isRendered ) {
+		isRendered = true;
+
+		const container = document.createElement( 'div' );
+
+		document.body.append( container );
+
+		ReactDOM.render(
+			<EditorDrawer />,
```
Comment (rotemee) [inline modules/notifications/assets/js/components/editor-on-button-clicked.js:14]: Dark mode? RTL?

**Review by rotemee — DISMISSED** [review]

**Review by arielk — APPROVED** [review]

**Pull request merged**